typescript/infra/scripts/check-deploy.ts

```diff
@@ -74,7 +76,15 @@ async function check() {
     multiProvider,
   );
   const routerConfig = core.getRouterConfig(envConfig.owners);
-  const ica = InterchainAccount.fromAddressesMap(chainAddresses, multiProvider);
+  const icaChainAddresses = objFilter(
```

Comment:
was there a specific reason for narrowing down the chain addresses in check-deploy?

Reply:
yeah - some chains don't have ICA routers deployed on them, and the SDK would complain that we'd pass in chainAddresses in there that didn't define any interchainAccountRouters

Reply:
similar deal for the RouterConfigs, we'd pass in a bunch of router configs in there that exceeded the chains in chainAddresses, and this caused issues